WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -196,19 +196,21 @@ def post_results
       if top333.empty? # If there was no 3x3x3 event.
         title = "Results of #{comp.name}, in #{comp.cityName}, #{comp.countryId} posted"
         body = "Results of the [#{comp.name}](#{competition_url(comp)}) are now available.\n\n"
-      elsif top333.length < 3
-        render html: "<div class='container'><div class='alert alert-danger'>Too few people competed in 333</div></div>".html_safe
-        return
       else
         title = "#{top333.first.personName} wins #{comp.name}, in #{comp.cityName}, #{comp.countryId}"
 
         body = "[#{top333.first.personName}](#{person_url top333.first.personId})"
         body += " won the [#{comp.name}](#{competition_url(comp)})"
-        body += " with an average of #{top333.first.to_s :average} seconds."
+        body += " with an average of #{top333.first.to_s :average} seconds"
 
-        body += " [#{top333.second.personName}](#{person_url top333.second.personId}) finished second (#{top333.second.to_s :average})"
+        if top333.length >= 2
+          body += ". [#{top333.second.personName}](#{person_url top333.second.personId}) finished second (#{top333.second.to_s :average})"
+          if top333.length >= 3
```

Comment:
Super nit: there's no way to be more than 3 of those, right? =)
Maybe `> 1` and `> 2`?